packages/web-components/src/menu/README.md

```diff
@@ -0,0 +1,110 @@
+# Menu
+
+> A Menu component for handling menus and menu items in a user interface.
+
+<br />
+
+## **Design Spec**
+
+[Link to Menu Design Spec in Figma](https://www.figma.com/file/xyz12345/Menu?node-id=2%3A476)
```

Comment:
Broken link, but I can't find the actual Menu spec in the folder with the rest of the specs.